src/runtime/providers/math.ts

```diff
@@ -63,6 +68,11 @@ function degreesToRadians(degrees:number){
 }
 
 class Add extends TotalFunctionConstraint {
+  resolveProposal(proposal, prefix) {
```

Comment:
At the moment you don't fail silently, you add them using Javascripts implicit rules :-) . I didn't want to introduce a breaking change to other peoples Eve code, but I agree that some thought needs to be given to implicit conversion. What about a combine aggregate?

I'll leave this as is for the pull unless told otherwise.